python/foglamp/services/core/server.py

```diff
@@ -115,10 +115,10 @@ class Server:
     rest_server_port = 0
     """ FogLAMP REST API port """
 
-    is_rest_server_http_enabled = False
+    is_rest_server_http_enabled = True
     """ a Flag to decide to enable FogLAMP REST API on HTTP on restart """
 
-    is_auth_required = True
+    is_auth_required = False
```

Comment:
Not needed, as when server starts it calls def rest_api_config (L#295) but yes for better readability we can.